internal/state/store_test.go

```diff
@@ -307,3 +306,67 @@ func TestABCIResponsesResultsHash(t *testing.T) {
 	require.NoError(t, err)
 	assert.NoError(t, proof.Verify(root, bz))
 }
+
+func TestLastABCIResponses(t *testing.T) {
+	//if the state store is empty
+	stateDB := dbm.NewMemDB()
+	stateStore := sm.NewStore(stateDB, false)
+	responses, err := stateStore.LoadABCIResponses(1)
+	require.Error(t, err)
+	require.Nil(t, responses)
+	fmt.Println(responses)
```

Comment:
```suggestion
```

left over debug?

Reply:
Yes sort of! I thought maybe we might want to keep it, its not necessary though

Reply:
`fmt.Println` is totally fine for active debugging, of course.


But for code that will be committed, `t.Log` or `t.Logf` is more appropriate in test code.

Typically I would only log for something that is actively going wrong; since this is an unconditional print, it's probably better removed.